package.json

```diff
@@ -226,22 +226,22 @@
         "@types/vscode": "^1.44.0",
         "@typescript-eslint/eslint-plugin": "^2.29.0",
         "@typescript-eslint/parser": "^2.29.0",
-        "eslint": "^6.8.0",
         "eslint-config-prettier": "^6.11.0",
         "eslint-plugin-prettier": "^3.1.3",
         "glob": "^7.1.6",
         "mocha": "^7.1.1",
-        "prettier": "2.0.5",
+        "prettier": "^2.0.5",
```

Comment:
Why do you remove dependency to eslint, `"eslint": "^6.8.0"`. As far as I understand it is required for prettier with current configuration works.